tests/Integration/SaveMappingsApiTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\Tests\Integration;
+
+use MediaWiki\Rest\RequestData;
+use MediaWiki\Tests\Rest\Handler\HandlerTestTrait;
+use MediaWikiIntegrationTestCase;
+use ProfessionalWiki\WikibaseRDF\Presentation\Rest\SaveMappingsApi;
+
+/**
+ * @covers \ProfessionalWiki\WikibaseRDF\Presentation\Rest\SaveMappingsApi
+ * @covers \ProfessionalWiki\WikibaseRDF\WikibaseRdfExtension
+ */
+class SaveMappingsApiTest extends MediaWikiIntegrationTestCase {
+	use HandlerTestTrait;
+
+	public function setUp(): void {
+		// TODO: Create Item
```

Comment:
See `SlotEntityContentRepositoryTest`. Should include `@group Database` on the class level and extend from `MediaWikiIntegrationTestCase`.